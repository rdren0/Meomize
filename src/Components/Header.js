import React from 'react';

function Header() {
  return (
    <div className='header-container'>
        <h1 className='header-title'>FlashCards!</h1>
        <h4> Categories: </h4>
        <button className="filter-buttons">RETURNS</button> 
        <button className="filter-buttons">SYNTAX</button> 
        <button className="filter-buttons">USE CASE</button>
        <button className="filter-buttons">ALL</button>

    </div>
  )
}
export default Header;