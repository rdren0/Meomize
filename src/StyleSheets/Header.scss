.filter-buttons:hover{
  background-color: #E1CE7A;
  color: black;
  font-weight: 700;
  border: 2px solid gainsboro;
  cursor: pointer;
}

.header-container{
  padding-top: 20px;
  margin-bottom: 0px;
  background-color: #1B3B6F;
  border-bottom: 1px solid gainsboro;
}

button{
  background-color: #21295C;
  color: #FFF;
  border: 2px solid gainsboro;
  border-radius: 15px;
  width: 100px;
  height: 40px;
  margin-bottom: 10px;
  margin-right: 10px;
  outline: none;
}


.categories-text{
  color: #21295C;
  margin-top: 0px;
  padding-top: 5px;
}

h1{
  color: #E1CE7A;
  font-size: 50px;
  margin-top: -5px;
}