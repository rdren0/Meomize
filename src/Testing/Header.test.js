import React from 'react';
import Header from '../Components/Header';
import { shallow } from 'enzyme';
import {configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

describe('Header', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <Header />
    )
  })

  it('should match the snapshot with all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });


});