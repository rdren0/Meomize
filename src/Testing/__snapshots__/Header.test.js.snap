// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot with all data passed in correctly 1`] = `
<div
  className="header-container"
>
  <h1
    className="header-title"
  >
    FlashCards!
  </h1>
</div>
`;
